src/app/rundown-overview/components/rundown-overview/rundown-overview.component.ts

```diff
@@ -46,8 +46,11 @@ export class RundownOverviewComponent implements OnInit, OnDestroy {
   }
 
   public openDeletionDialog(basicRundown: BasicRundown): void {
-    this.dialogService.createConfirmDialog('Delete Rundown?', `Are you sure you want to delete the Rundown "${basicRundown.name}"?\n\nPlease note: This action is irreversible!`, 'Delete', () =>
-      this.deleteRundown(basicRundown.id)
+    this.dialogService.createConfirmDialog(
+      $localize`rundown-overview-component.deletion-dialog-tittle:Delete Rundown?`,
+      $localize`rundown-overview-component.deletion-dialog-question:Are you sure you want to delete the Rundown "${basicRundown.name}"?\n\nPlease note: This action is irreversible!`,
```

Comment:
I'm in favour for the latter

Reply:
I am afraid that the syntax won't be able to be applied since I am getting an error saying that TypeScript is expecting a TemplateStringsArray type, and not a plain string.